### PR TITLE
clean up different profiles

### PR DESCRIPTION
**Pull request by richardwxn**
some unnecessary extra settings in helm values part are removed.

### CODE REVIEW

File: operator/data/profiles/demo.yaml [hunk 1]
```diff
@@ -105,12 +104,5 @@ spec:
       istio-ingressgateway:
         autoscaleEnabled: false
 
-    grafana:
-      enabled: true
```
Comment (howardjohn) [inline operator/data/profiles/demo.yaml]: ??

**Review by howardjohn — COMMENTED** [review]
I am a bit confused. the default is only prometheus addon is enabled, the others are disabled, but you seem to be changing this, rather than just "cleaning up". Am I missing something?

File: operator/data/profiles/demo.yaml (hunk 1 above)
Comment (richardwxn) [reply operator/data/profiles/demo.yaml]: this part is already defined in the addonComponents part above

**Comment by richardwxn** [comment]
@howardjohn I am actually removing duplicate entries appear in both IOP and helm values path. I keep those in IOP and remove duplicate from helm values path

**Review by howardjohn — APPROVED** [review]
got it. thanks for the clarification

**Pull request merged**